tasks/write-to-a-windows-event-log/src/main.rs

```diff
@@ -0,0 +1,44 @@
+#[cfg(target_os = "windows")]
+mod bindings {
+    ::windows::include_bindings!();
+}
+
+const TOKEN_READ: u32 = 0x00020008;
+const EVENTLOG_WARNING_TYPE: u16 = 2;
+
+#[cfg(target_os = "windows")]
+fn main() {
+    use bindings::windows::win32::security::*;
```

Comment:
Let's avoid using wildcard imports for these. I like the example from the `windows` crate README where the import block mirrors the structure in build.rs.